addons/point_of_sale/static/src/js/models.js

```diff
@@ -351,16 +344,19 @@ class PosGlobalState extends PosModel {
     cashierHasPriceControlRights() {
         return !this.config.restrict_price_control || this.get_cashier().role == 'manager';
     }
+    _onReactiveOrderUpdated(order) {
+        order.save_to_db();
+    }
```

Comment:
You can remove this method. Just do the same override as you did with `createAutomaticallySavedOrder` for automatically tracking the `print diffs` in the order.

Reply:
Furthermore, kindly rename `createAutomaticallySavedOrder` to `createReactiveOrder`.

Reply:
I'm mistaken here. You should include the `this.ordersToUpdateSet.add(order);` in `_onReactiveOrderUpdated` because you want to update it whenever an order is changed.